tests/unit/jobs_test.yaml

```
@@ -0,0 +1,47 @@
+---
+suite: Custom Annotations
```

```suggestion
suite: Extra Volumes
```

Could we rename this file to `extra_volumes_test.yaml`?

Each file tests one feature across all applicable templates. We can add more extra_volumes tests in another PR for all the deployments that use this too.

---

Both file and suite name have been updated